src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -36,15 +36,55 @@
         </v-btn>
       </div>
 
+      <div v-else-if="showAlterNowButton" class="my-1">
+        <div v-if="isAllowAlterOnline" class="d-flex justify-center alter-now-btn">
+          <v-btn
+            class="alter-now-external-btn"
+            v-if="isOpenExternal"
+            min-width="20rem"
+            :disabled="disabled"
+            @click="$emit('goToCorpOnline')"
+          >
+            <strong>Alter Now</strong>
+            &nbsp;
+            <v-icon small>mdi-open-in-new</v-icon>
+          </v-btn>
+          <v-btn
+            v-else
+            class="alter-now-btn"
+            min-width="20rem"
+            :disabled="disabled"
+            @click="$emit('goToEntityDashboard')"
+          >
+            <strong>Alter Now</strong>
+          </v-btn>
+        </div>
+
+        <div v-else class="contact-registries">
+              To complete this alteration, please contact us at:
+              <p>
+                <br/>
```

Comment:
Don't use this for whitespace in this paragraph. Instead, use class mt-x or pt-x in the paragraph element itself.